include/model/CHierarchicalResultsNormalizer.h

```diff
@@ -44,6 +45,19 @@ struct MODEL_EXPORT SNormalizer {
     //! Compute a checksum for this object.
     uint64_t checksum() const;
 
+    void debugMemoryUsage(const core::CMemoryUsage::TMemoryUsagePtr& mem) const {
```

Comment:
I suspect declaring inline in a `MODEL_EXPORT` class is causing link issue on windows.